runtime.go

```diff
@@ -26,13 +27,19 @@ func PrettyJson(a interface{},msg ...string) {
 	if ok {
 		fmt.Println("__FILE__", file, "__LINE__", ln)
 		if a!=nil {
+			if reflect.TypeOf(a).String() == "[]uint8" {
```

Comment:
Are you trying to check for []byte? or []uint8 ?

Reply:
I am trying to check []byte, but it is just an alias of []uint8. 
- https://play.golang.org/p/fzmQfw8AxxH
- https://stackoverflow.com/questions/37770005/why-is-there-no-byte-kind-in-the-reflect-package